operations/iuf/workflows/management_rollout.md

```diff
@@ -21,6 +21,15 @@ Once this step has completed:
 
 - Host firmware has been updated on management nodes
 
+Note: Before proceeding with `management-nodes-rollout` iSCSI SBPS configuration has to be done where worker nodes need
+to be configured as iSCSI targets for iSCSI SBPS. Unlike in CSM 1.6.0, where all worker nodes were configured as iSCSI
+targets by default without requiring any manual steps, in CSM 1.7.0, selective worker node personalization is introduced
```

Comment:
This section also needs to mention that if they DO choose to create the group and limit the workers being configured, then they will have manual DNS steps to perform, right? Making the group only takes care of part of it, I thought.